packages/router/src/params.tsx

```diff
@@ -9,7 +9,9 @@ export interface ParamsContextProps {
 
 export const ParamsContext = createNamedContext<ParamsContextProps>('Params')
 
-export const ParamsProvider: React.FC<{}> = ({ children }) => {
+export const ParamsProvider: React.FC<Record<string, unknown>> = ({
```

Comment:
Basically don't use `{}`.

Reply:
I'm open to changing this completely... I was just trying to get rid of the errors

Reply:
Here @Tobbe 

![Screenshot 2021-03-23 at 11 16 00](https://user-images.githubusercontent.com/1521877/112138248-30e60380-8bc9-11eb-8afe-ef8fc95f6757.png)

Reply:
If I could run the linter locally, this would be much easier... But I think the message is misleading. It thinks I'm using {} as an empty object, when I'm actually using it as an empty interface.

Reply:
Try without the generics stuff, it should default to {}

Reply:
Done ✅